opendbc/dbc/generator/SConscript

```diff
@@ -0,0 +1,18 @@
+import os
+from opendbc.dbc.generator.generator import (generated_suffix, generator_path, opendbc_root, \
+                                  get_python_generator_scripts, get_source_dbc_files)
+Import('env')
+
```

Comment:
this looks way too complicated. why not just glob everything in generator/*?

Reply:
not all `.dbc` within generator dir being the sources for the generated DBCs :(https://github.com/commaai/opendbc/blob/8cf033fa3ee6191c10e5d63702f8e3211095a29b/opendbc/dbc/generator/generator.py#L57)

glob.glob(f"{generated_path/*/*.dbc}" with added conditional statements and replacing the `.dbc` suffix with `_generated.dbc` would be the same as what you referenced

it's easier abstract out the search functions to get the path to generated dbc to delete those when run with `scons -c`